packages/query/src/graphql/graphql.module.ts

```diff
@@ -123,7 +120,18 @@ export class GraphqlModule implements OnModuleInit, OnModuleDestroy {
         options.replaceAllPlugins.push(options.appendPlugins.pop());
       }
     }
-    const schema = await this.buildSchema(dbSchema, options, SCHEMA_RETRY_NUMBER);
+
+    if (!argv['disable-hot-schema']) {
+      const pgClient = await this.pgPool.connect();
+      await pgClient.query(`LISTEN "${dbSchema}._metadata.hot_schema"`);
+
+      pgClient.on('notification', (msg) => {
+        if (msg.payload === 'schema_updated') {
+          void this.schemaListener(dbSchema, options);
```

Comment:
I think we should exit if this fails. In the catch for schemaListener, can we log the error and process.exit